packages/node_modules/@ciscospark/widget-space/src/actions.js

```diff
@@ -1,4 +1,22 @@
+import {isUuid} from '@ciscospark/react-component-utils';
+
+export const FETCH_SPACE_ERROR = `widget-space/FETCH_SPACE_ERROR`;
```

Comment:
Oh, I think i misinterpreted this specific case. This is actually the specific error that gets stored when the fetching a space errors out.

I was imagining generic keyed store for any error, and actions that pushed errors into that store. It could then be imported into any widget, and you could just check flags on that store to see if there's an error or not. Displaying the error would still be up to the widget itself, but there would be a global state.

That would make it so that when we start composing the widgets, the top level widget could read errors stored by the lower level widgets and react to them.